weaviate/collection/classes.py

```diff
@@ -450,7 +450,7 @@ def type_to_dict(model: Type["BaseProperty"]) -> List[Dict[str, Any]]:
         reference_fields = model.get_ref_fields(model)
         properties.extend(
             {
-                "name": _capitalize_first_letter(name),
+                "name": name,
```

Comment:
Are the capitalizations here and on L439 handled somewhere else and are thus redundant now? Perhaps on L428 of `collection_model.py`?

Reply:
see other comment